nextflow.config

```diff
@@ -27,6 +27,10 @@ params {
     // Alignment
     aligner                    = 'bwamem2'
 
+    // Annotation
+    vep_genome                 = 'GRCh37'
```

Comment:
We should try to use the already given `--genome` parameter instead. Doesn't make sense to use align to one genome and annotate it with another. Might be that you have to parse that parameter but looking at the `genomes.config` you should be able to use it straight out of the box.